doc/Pacemaker_Explained/en-US/Ch-Multi-site-Clusters.txt

```diff
@@ -216,23 +215,44 @@ To revoke a ticket from this site:
 
 [IMPORTANT]
 ====
-If you are managing tickets manually. Use the `crm_ticket` command with
-great care as they cannot help verify if the same ticket is already
+If you are managing tickets manually, use the `crm_ticket` command with
+great care, because it cannot check whether the same ticket is already
 granted elsewhere. 
-
 ====
 
 
 === Granting and Revoking Tickets via a Cluster Ticket Registry ===
 
-==== Booth ====
-Booth is an implementation of `Cluster Ticket Registry` or so-called
-`Cluster Ticket Manager`.
+We will use https://github.com/ClusterLabs/booth[Booth] here as an example of
+software that can be used with pacemaker as a Cluster Ticket Registry.  Booth
+implements the
+http://en.wikipedia.org/wiki/Paxos_%28computer_science%29['Paxos'] lease
+algorithm to guarantee the distributed consensus among different
+cluster sites, and manages the ticket distribution (and thus the failover
+process between sites).
```

Comment:
Booth has switched to raft algrorithm:
http://en.wikipedia.org/wiki/Raft_%28computer_science%29